docs/release-notes/release-notes-0.17.0.md

```diff
@@ -13,14 +13,27 @@
   the entire retribution struct. This reduces the amount of data that needs to 
   be held in memory. 
 
+## RPC
+
+* The [UpdateNodeAnnouncement](https://github.com/lightningnetwork/lnd/pull/7568)
+  API can no longer be used to set/unset protocol features that are defined by 
+  LND.  
+
+  Custom node announcement feature bits can also be specified in config using 
+  the `dev` build tag and `--protocol.custom-nodeann`, `--protocol.custom-init` 
+  and `--protocol.custom-invoice` flags to set feature bits for various feature
```

Comment:
Since they're behind the `dev` built tag, I think we should not add them. That's also why the `CI / sample configuration check` check did not trigger on them.